README.md

```diff
@@ -3,23 +3,12 @@
         <img src="https://yiisoft.github.io/docs/images/yii_logo.svg" height="100px">
     </a>
     <a href="https://getbootstrap.com/" target="_blank" rel="external">
-        <img src="https://v4-alpha.getbootstrap.com/assets/brand/bootstrap-solid.svg" height="100px">
+        <img src="https://v4-alpha.getbootstrap.com/assets/brand/bootstrap-solid.svg" height="80px">
```

Comment:
```suggestion
        <img src="https://v4-alpha.getbootstrap.com/assets/brand/bootstrap-solid.svg" height="100px">
```

Reply:
I kept all Yii logos at 100px to give more emphasis to the second logo, at 80px.
Yii is the star of the show, not a supporting character.

Reply:
It looks better when sizes are equal.